src/microbe_stage/editor/MicrobeEditorGUI.cs

```diff
@@ -1401,7 +1401,7 @@ private void OnRigidityChanged(int value)
 
     private void OnColorChanged(Color color)
     {
-        editor.Colour = color;
+        editor.CurrentSpecies.Colour = color;
```

Comment:
This breaks a feature.

After this change the live membrane colour preview no longer works.